options/locale/locale_en-US.ini

```
@@ -2411,6 +2411,7 @@ branch.included_desc = This branch is part of the default branch
 branch.included = Included
 branch.create_new_branch = Create branch from branch:
 branch.confirm_create_branch = Create branch
+branch.warning_raname_default_branch = You are trying to rename default branch.
```

```suggestion
branch.warning_rename_default_branch = You are trying to rename the default branch.
```

And the typo `raname` needs to be fixed elsewhere as well.